docker/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM php:7.4-apache
+
+ARG BRANCH=stable
+
+COPY --chmod=755 ./entrypoint.sh /usr/local/sbin/entrypoint
+
+VOLUME /overlay/storage
+
+# setup DokuWiki
+RUN mkdir -p /overlay/original && \
```

```suggestion
RUN set -x && \
    mkdir -p /overlay/original && \
```

I like the shell to show what commands are executed, eases debug and has no downside for the final image.